pkg/provider/gitlab/detect.go

```diff
@@ -37,7 +37,7 @@ func (v *Provider) Detect(req *http.Request, payload string, logger *zap.Sugared
 
 	switch gitEvent := eventInt.(type) {
 	case *gitlab.MergeEvent:
-		if provider.Valid(gitEvent.ObjectAttributes.Action, []string{"open", "update", "reopen"}) {
+		if provider.Valid(gitEvent.ObjectAttributes.Action, []string{"open", "reopen"}) {
```

Comment:
do we cover if a new commit is pushed? not sure which event that would be ..Push?

Reply:
i messed my commits :) but yeah you are right, i think I got it figured now,
 but will send everything on #1429